tests/behat/features/ra_select-raa_use_raa.feature

```diff
@@ -9,22 +9,22 @@ Feature: A RAA can only manage R RA(A)'s on the promotion page
     Given a user "jane-d-user" identified by "urn:collab:person:institution-d.example.com:jane-d-user" from institution "institution-d.example.com" with UUID "00000000-0000-4000-a000-000000000002"
     And the user "urn:collab:person:institution-d.example.com:jane-d-user" has a vetted "yubikey" with identifier "00000005"
 
-    And institution "institution-a.example.com" can "use_ra" from institution "institution-a.example.com"
     And institution "institution-a.example.com" can "use_raa" from institution "institution-a.example.com"
+    And institution "institution-a.example.com" can "use_raa" from institution "institution-d.example.com"
     And institution "institution-a.example.com" can "select_raa" from institution "institution-a.example.com"
     And institution "institution-a.example.com" can "select_raa" from institution "institution-d.example.com"
 
-    And institution "institution-d.example.com" can "use_ra" from institution "institution-d.example.com"
+    And institution "institution-d.example.com" can "use_raa" from institution "institution-a.example.com"
     And institution "institution-d.example.com" can "use_raa" from institution "institution-d.example.com"
-    And institution "institution-d.example.com" can "select_raa" from institution "institution-d.example.com"
+#    And institution "institution-d.example.com" can "select_raa" from institution "institution-d.example.com"
```

Comment:
(C|S)hould be removed completely?

Reply:
I deliberately left it there so the test would be better readable in the end.

Reply:
Hmm, in that case an inverted step would be more explaining then a commented out step. Or give it some additional context by adding a comment. To me this does not add any readability.

Reply:
Done